### DISCUSSION TITLE
Change update interval

### DISCUSSION THREAD

**Discussion by Koning-Martin** [leeyuentuen/alfen_wallbox#56] [Ideas]
Hereby a wish for the feature backlog:

The integration now updates every 30 seconds all the values. It would be nice if it is possible to make this period shorter. I don’t know how often would be possible without overflowing the CP with requests. Five seconds sounds really good, ten seconds sound as a good alternative.

**Comment by leeyuentuen** [comment]
will try to test them out

**Comment by leeyuentuen** [comment]
@Martin-Yoshi  you can take a look at this beta version: https://github.com/leeyuentuen/alfen_wallbox/releases/tag/2.6.9

**Reply by Koning-Martin** [reply]
I updated to 2.6.9, the power (watt) seems to work every 5 sec. Other values like frequency are acting weird (like not updating at all). I will test more, maybe I’m to impatient. I will test more this weekend and share my experience! :-)

**Comment by Koning-Martin** [comment]
Quick first impression:
- The wattage sensor is the only one updating every 5 seconds. 
- The frequency sensor is not updating at all (herz).
- All the other sensors are updating between 5 and 16 seconds. Example: Voltage sensor renews after 7 seconds, next time after 14, then after 5, then after 16 etc. I can’t find a pattern in it. 

Can you do something with this feedback or do you need more specific info?

**Comment by leeyuentuen** [comment]
the value will update if it is not the same as the previous change

so if you have 0 then it will be not update untill the value has been changed. for the hertz before it was 2 digits e.g 50.20 but now it is only 0 digit after the comma, so it became 50. so if it keeps between  50 and 50.9, you will not see an update. maybe we change them to 1 digit instead of 2 digits as before

same like voltage, if it keeps 230V, then you won't see the difference.

this is how HA works, if you check other things like the ip or simcard number, they are not update since the reboot of HA

**Comment by leeyuentuen** [comment]
reason why i've change the digits, to prevent to have too much log in the recorder, that will make the DB to become big

**Reply by Koning-Martin** [reply]
Nice! Thank you for your explanation, makes perfect sense to me. I will keep testing this weekend! 😊

**Reply by leeyuentuen** [reply]
i'm now just curious if this still keep the reboot issue stable. because we made here alot more calls. 6times more then before from 30sec to 5sec

and each time it made around 5-6 calls

**Reply by Koning-Martin** [reply]
So far everything seems to run fine! Installed it 20 hours ago, no reboot (yet). The sensors keep updating, so that’s good. I will keep testing. 😎

**Comment by Koning-Martin** [comment]
Small update: Integration was updated last Friday night (around 19:00) to version 2.6.9. So now it’s online for 52 hours, I don’t see any problems or reboots so far… I will keep you posted.

**Comment by leeyuentuen** [comment]
still need to test this issue before release to public

https://github.com/leeyuentuen/alfen_wallbox/issues/60

**Comment by leeyuentuen** [comment]
@Martin-Yoshi  this has been fixed I suppose, on that way, I'll close this discussion.
